Makefile

```diff
@@ -89,15 +82,18 @@ ct/ct.o ct/_ctcheck.o: ct/ct.h ct/internal.h
 
 $(TOFILES): $(HFILES) ct/ct.h
 
-CLEANFILES:=$(CLEANFILES) ct/_* ct/*.o
+CLEANFILES+=$(wildcard ct/_* ct/*.o)
 
-ifneq ($(shell ./verc.sh),$(shell cat vers.c 2>/dev/null))
-.PHONY: vers.c
-endif
 vers.c:
 	./verc.sh >vers.c
 
 doc/beanstalkd.1 doc/beanstalkd.1.html: doc/beanstalkd.ronn
 	ronn $<
 
 freebsd.o: darwin.c
+
+.PHONY: all install clean check bench
```

Comment:
I find it easier to have PHONY section near the target. Even thought it's more lines, it's easier to manage and understand.

Reply:
A lot of people find it clutter and prefer it at the end where it's easier to ignore.  But I've made the change.

Reply:
I see your point. But I want to keep it on the safe, conservative side for a while. I appreciate your feedback.